platform/viewer/src/index.js

```diff
@@ -27,26 +27,27 @@ import OHIFDicomHtmlExtension from '@ohif/extension-dicom-html';
 import OHIFDicomMicroscopyExtension from '@ohif/extension-dicom-microscopy';
 import OHIFDicomPDFExtension from '@ohif/extension-dicom-pdf';
 
-// Default Settings
+/*
+ * Default Settings
+ */
 let config = {};
-const appDefaults = {
-  routerBasename: '/',
-};
 
 if (window) {
   config = window.config || {};
-  config.extensions = [
+}
+
+const appProps = {
+  config,
+  defaultExtensions: [
     OHIFVTKExtension,
     OHIFDicomHtmlExtension,
     OHIFDicomMicroscopyExtension,
     OHIFDicomPDFExtension,
-  ];
-}
-
-const appProps = Object.assign({}, appDefaults, config);
+  ],
```

Comment:
This is a breaking change. We need to document this and do a major version bump.